### ISSUE TITLE
[FEATURE] Playlist/Folder

### ISSUE THREAD

**Issue by theoadam9** [eni9889/YT-PP-Issues#5]
I option to create a folder to place certain videos or music.

For example:
• Music Genres
• Video Series
• Parts of a video

Thanks,
Theo.H

**Comment by D3m3rion** [comment]
Yeah, would be cool to have an option to create folders in the download section

**Comment by JMccormick264** [comment]
I like that idea. A way to keep things more organized

**Comment by theoadam9** [comment]
Thanks, I think really would help out a lot.
